fast64_internal/f3d/f3d_material_nodes.py

```diff
@@ -5,1123 +5,1122 @@
 from .f3d_enums import *
```

Comment:
this file is deleted in the #109 diff, may as well not format it?

Reply:
i formatted it since that original format commit had done that and i was avoiding conflicts haha